src/schemes/boundary/dummy_particles/dummy_particles.jl

```diff
@@ -306,15 +272,22 @@ end
     boundary_model.pressure[particle] = max(boundary_model.state_equation(density), 0.0)
 end
 
+reset_wall_velocity!(viscosity, system) = system
+
+@inline function reset_wall_velocity!(viscosity::ViscosityAdami, system)
+    set_zero!(system.cache.wall_velocity)
+    return system.cache
+end
```

Comment:
```suggestion
@inline function reset_wall_velocity!(system, viscosity::ViscosityAdami)
    set_zero!(system.cache.wall_velocity)
    return system
end
```
`system` is the argument that is modified.